generic/tests/cfg/kdump.cfg

```diff
@@ -29,10 +29,14 @@
         kdump_propagate_cmd = "kdumpctl propagate"
         kdump_restart_cmd = "systemctl restart kdump.service"
     RHEL.8:
+        x86_64:
+            kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=auto"
+            kdump_enable_cmd = "systemctl restart kdump.service"
         ppc64le, ppc64:
             kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=512M@64M"
             kdump_restart_cmd = "systemctl restart kdump.service"
     RHEL.9:
+        kernel_param_cmd = "grubby --update-kernel=`grubby --default-kernel` --args=crashkernel=1G-4G:192M,4G-64G:256M,64G-:512M"
```

Comment:
I would recommend to remove `RHEL.9` and move this line up to L#18, as it is for not only RHEL.9 but all later version. Otherwise you will need to add versions here for all later release.